packages/theme/getStore.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import urlJoin from 'url-join';
+
+interface StoreConfig {
+  backendUrl: string
+}
+
+/* eslint-disable camelcase */
+export interface StoreAttr {
+  type: string
+  id: string
+  attributes: {
+    name: string
+    url: string
+    meta_description: string | null
+    meta_keywords: string | null
+    seo_title: string | null
+    default_currency: string
+    supported_currencies: string
+    default_locale: string
+    supported_locales: string
+  }
+}
+interface IStore {
+  data: StoreAttr
+}
+interface Store {
+  name: string
+  url: string
+  metaDescription?: string
```

Comment:
Perhaps, you could sort object properties by its optionality - required first, then, optionals. This will improve readability.